src/test/com/wolfssl/test/WolfSSLContextTest.java

```diff
@@ -644,13 +644,6 @@ public void test_WolfSSLContext_setMinECCKeySize() {
                 fail("setMinECCKeySize should fail with negative key size");
             }
 
-            /* key length not % 8 should fail */
-            ret = ctx.setMinECCKeySize(255);
```

Comment:
Why was this test case removed?

Reply:
Found the answer in the other pull request (https://github.com/wolfSSL/wolfssljni/pull/258):

```
Also includes one ECC test fix which causes JUnit tests to fail after a recent native wolfSSL ECC key size adjustment.
```